tests/test_routes.py

```diff
@@ -522,6 +534,112 @@ def test_delete_address_invalid_request(self):
         resp = self.client.delete(f"{BASE_URL}/{customer.id}/addresses/{address.address_id}") #second time
         self.assertEqual(resp.status_code, status.HTTP_204_NO_CONTENT)
 
+
+    ##########################################
+    # UPDATE CUSTOMER
+    ##########################################
+    def test_update_customer(self):
+        """It should Update an existing Customer"""
+        # create a Customer to update
+        test_customer = CustomerFactory()
+        response = self.client.post(BASE_URL, json=test_customer.serialize())
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        # update the Customer
+        dummy_email =  "User759@gmail.com";
+        updated_customer = response.get_json()
+        logging.debug(updated_customer)
+        updated_customer["email"] = dummy_email
```

Comment:
Can we add updations of all other customer fields as well (except the ID)? Can do it in same method or different.
This will be just for completion, since we did all fields for CREATE as well as READ.

Reply:
Sure

Reply:
Changes have been pushed, so marking this as resolved.